source/TagsWindow.cpp

```diff
@@ -20,18 +20,25 @@
 #include <Entry.h>
 #include <Message.h>
 #include <View.h>
+#include <Catalog.h>
+#include <TranslationUtils.h>
 
 #include <stdio.h>
 #include <stdlib.h>
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "TagsWindow"
+
 // Constants
 #define BUTTON_OK 		'btok'
-#define BUTTON_UNDO 	'btun'
+#define BUTTON_CANCEL 	'btcn'
 #define TAGS_CLOSE		'_tgc'
 
 
+
 TagsWindow :: TagsWindow(BMessage *fSaveMessage, BHandler *handler)
-		   : BWindow (BRect(300,300,700,550),"Set Tags for this note",B_TITLED_WINDOW, B_NOT_RESIZABLE){
+		   : BWindow (BRect(300,300,700,450), B_TRANSLATE("Set Tags for this note")
+		   ,B_TITLED_WINDOW, B_NOT_RESIZABLE){
 
```

Comment:
The tags window should also use BLayoutBuilder and all widgets the layout-aware versions without the BRect parameter.